src/models/Order.js

```diff
@@ -4,8 +4,10 @@ module.exports = (sequelize) =>
   sequelize.define('Order', {
     status: {
       type: DataTypes.ENUM,
-      // TODO check this values
-      values: ['active', 'pending', 'deleted'],
+      //Confirmed: the order has been created, Processing: the shop is preparing the order to ship it
+      //Shipped: the order has been sent to the given address, Delivered: the order has reached the destination and recepted by a person.
+      //Canceled: the order has been canceled
+      values: ['confirmed', 'processing', 'shipped', 'delivered', 'canceled'],
```

Comment:
I think we need here a 'awaiting payment' or something like that, like the first 'step' ? or are we going to create the order after the payment? 
the would be :
1. choose the products and add them to the cart
2. click 'pay' 
3. add all the data to create the payment (generate the order 'awaiting payment')
4. pay with 'credit card' or sth like that (update the order to confirmed? or processing?